### PR TITLE
Improve instructions for running eVaka locally

### PR DESCRIPTION
**Pull request by kustaahuhtala**
#### Summary

Improve instructions for running eVaka on local development environment.

#### Dependencies

None

#### Testing instructions

Please read through and check for any typos. Also, please double-check versions I've added.

#### Checklist for pull request creator

- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The branch has been rebased against master before the PR was created

#### Checklist for pull request reviewer (copy to review text box)

```
- [ ] All changes in all changed files have been reviewed
- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The PR branch has been rebased against master and force pushed if necessary before merging
```

### CODE REVIEW

File: compose/README.md [hunk 1]
```diff
@@ -33,12 +31,36 @@ environment locally. You can install them through your development
...
   Kit, version 11+. We recommend using OpenJDK implementation of JSR 384.
 - [Docker](https://docs.docker.com/get-docker/) – Docker is an open platform for developing, shipping, and running applications.
 
+
+### Required tooling
+
+eVaka uses [PM2](https://pm2.keymetrics.io/) for running multiple
+services in parallel. PM2 is an opinionated choice for
+development process in eVaka. Installing PM2 is not strictly
+mandatory. You can also run every sub-project in separate processes or
```
Comment (mikkopiu) [inline compose/README.md:50]:
```suggestion
mandatory. You can also run every sub-project separately or
```
They're always technically in separate processes.

File: compose/README.md [hunk 2]
```diff
@@ -33,12 +31,36 @@ environment locally. You can install them through your development
...
+build Docker images and run the stack using Docker (please refer to the
+[documentation for this](#running-the-full-stack-for-e2e-tests)).
```
Comment (mikkopiu) [inline compose/README.md:52]: Maybe just simplify:

```suggestion
build Docker images and [run the stack using Docker](#running-the-full-stack-for-e2e-tests)).
```

File: compose/README.md [hunk 3]
```diff
@@ -33,12 +31,36 @@ environment locally. You can install them through your development
...
+
+Install PM2 by running following command
+
+```bash
+npm install -g pm2
+```
+
+You will also need `nc/netcat` (Arbitrary TCP and UDP connections and listens) utility.
```
Comment (mikkopiu) [inline compose/README.md:60]:
```suggestion
You will also need the `nc/netcat` (Arbitrary TCP and UDP connections and listens) utility.
```

File: compose/README.md [hunk 4]
```diff
@@ -119,6 +141,12 @@ Access the frontends at
 
 ## Troubleshooting
 
+### Docker
+
+If you are having troubles starting Docker, make sure the user is added
+to `docker` group. See [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/) 
+for further reference.
+
```
Comment (mikkopiu) [inline compose/README.md:149]: Is this really necessary to document here? This is pretty generic Docker installation stuff

File: frontend/README.md [hunk 5]
```diff
@@ -92,14 +91,14 @@ cd packages/employee-frontend
 ICONS=free yarn dev
 ```
 
-### Using pro icons
+### Using professional icons
```
Comment (mikkopiu) [inline frontend/README.md:94]: Please fix all the links pointing to this title in:
```
compose/README.md:With [pro icons](../frontend/README.md#using-pro-icons)
frontend/packages/employee-frontend/README.md:so you need to [configure that first](../README.md#using-pro-icons)
frontend/packages/enduser-frontend/README.md:so you need to [configure that first](../README.md#using-pro-icons)
```

**Review by mikkopiu — CHANGES_REQUESTED** [review]

File: compose/README.md (hunk 4 above)
Comment (kustaahuhtala) [reply compose/README.md]: I guess not 😄

File: frontend/README.md (hunk 5 above)
Comment (kustaahuhtala) [reply frontend/README.md:94]: I think they're OK now.

**Review by mikkopiu — APPROVED** [review]
- [x] All changes in all changed files have been reviewed
- [x] The code is consistent with the existing code base
- ~[ ] Tests have been written for the change (e2e, integration tests)~
- [x] The change has been tested locally
- ~[ ] The change conforms to the UX specifications~
- [x] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [x] The PR branch has been rebased against master and force pushed if necessary before merging

**Pull request merged**